src/loadtests/functionaltests/main.py

```diff
@@ -1,28 +1,27 @@
-import sys
+import re
+import os
 import time
 import logging
 import threading
+from urllib.parse import urlparse
 from typing import Optional
 
 from gevent import monkey
 # monkey-patch the monkey patch because it prevents debugging and has no use here
 monkey.patch_all = lambda *args, **kwargs: ()
 
 from prometheus_client import start_http_server, Gauge
-
 from loadtests.functionaltests.functionalTester import FunctionalTester
+from loadtests.utils.envhandler import get_environment_var
 
-
-TIMEINTERVAL_SEC = 300
-PROMETHEUS_PORT = 9000
-HOSTS = [
-    ('staging_niedersachsen', 'https://staging.niedersachsen.dbildungscloud.org')
-]
+TIMEINTERVAL_SEC = get_environment_var('TIMEINTERVAL_SEC', int, default=300)
+PROMETHEUS_PORT = get_environment_var('PROMETHEUS_PORT', int, default=9000)
+TARGET_URL = get_environment_var('TARGET_URL')
```

Comment:
Jetzt gibt es zwei Varianten sich zu konfigurieren, was mir etwas inkonsistent vorkommt:
- Env Variable direkt mittels get_environment_var() laden
- Variable aus loadtests.config.Config benutzen

Letztere Variante hat den Vorteil, dass die (teils gemeinsame) Konfiguration an einem Ort stattfindet und abgebrochen wird, falls unvollständig, samt Auflistung fehlender Env Variablen.

Reply:
Moved to Config class